ci/jjb/jobs/pulp-vault.yaml

```diff
@@ -0,0 +1,233 @@
+        - string:
+            name: PRE_PYTEST_ARGS
+            default: -v --color=yes --pyargs pulp_2_tests.tests.platform.api_v2.test_login
+            description: Pytest sanity check (quick test) to run before upgrading the O.S.
+        - string:
+            name: POST_PYTEST_ARGS
+            default: -v --color=yes --junit-xml=junit-report.xml --pyargs pulp_2_tests.tests
+            description: Pytest command to run after the upgrade of O.S.
+
+
+        - string:
+            name: SMASH_REPO
+            default: git+https://github.com/PulpQE/pulp-smash.git@master#egg=pulp-smash
+        - string:
+            name: TEST_REPO
+            default: git+https://github.com/PulpQE/pulp-2-tests.git@master#egg=pulp-2-tests
+
+
+        - string:
+            name: PULP_SMASH_PULP_VERSION
+            default: '2.19.1'
+            decription: The Pulp version Pulp Smash will consider when running tests after the upgrade. Specify the .Z stream if needed.
+        - string:
+            name: PULP_SMASH_PULP_USERNAME
+            default: admin
+        - string:
+            name: PULP_SMASH_PULP_PASSWORD
+            default: admin
+        - choice:
+            name: PULP_SMASH_AMQP_BROKER_SERVICE
+            choices:
+                - qpidd
+                - rabbitmq
+        - choice:
+            name: PULP_SMASH_API_SCHEME
+            choices:
+                - https
+                - http
+        - bool:
+            name: PULP_SMASH_API_VERIFY
+            default: false
+    
+    properties:
+      - qe-ownership
+    scm:
+        # - pulp-ci-github
+        - git:
+            url: https://github.com/rochacbruno/pulp-ci.git
+            branches:
+                - '*/vaultjob'
+            skip-tag: true
+    wrappers:
+        - ansicolor
+        - timeout:
+            # pulp-smash usually takes about an hour or two, so time it out
+            # after 7:40 hours. This gives room to some tasks time out.
+            timeout: 460
+            abort: true
+        - timestamps
+        - config-file-provider:
+            files:
+                - file-id: rhn_credentials
+                  variable: RHN_CREDENTIALS
+        - jenkins-ssh-credentials
+        - credentials-binding:
+            - text:
+                credential-id: UPSHIFTPASSWORD
+                variable: UPSHIFTPASSWORD
+    builders:
+        - fix-hostname
+        - shell:
+            !include-raw:
+              # Creates parameters.txt
+              - 'create_upshift_instance.sh'
+        - inject:
+            # injects PULP_HOSTNAME
+            properties-file: parameters.txt
+        - shell: |
+            # Useful for debugging
+            getent passwd "$(id -u)"
+
+            # Setup ssh config and private key
+            mkdir ~/.ssh/controlmasters
+            
+            cat > ~/.ssh/config <<EOF
+            Host ${PULP_HOSTNAME}
+                User cloud-user
```

Comment:
The `cloud-user` was just for RHEL. Fedora will use `fedora`. It will be different depending on the base template. :(

Reply:
This vault job will always run RHEL